### DISCUSSION TITLE
Why doesn't current line in the current line when on the relativenumber configuration?

### DISCUSSION THREAD

**Discussion by lingdor** [JetBrains/ideavim#779] [General]
Why does current line number is not current line number when on the relativenumber configuration?
![image](https://github.com/JetBrains/ideavim/assets/57614815/4673d1e7-2ceb-483f-831a-d770ea5945d3)
in that image, I want the current number is "3", like that:
![image](https://github.com/JetBrains/ideavim/assets/57614815/55fb5dae-ff0f-44bc-9a88-f6185387c9c8)
how can i do it ,thanks

**Comment by citizenmatt** [comment]
You need to `set number` as well as `set relativenumber`

**Reply by lingdor** [reply]
Thanks
